src/Registry.chpl

```diff
@@ -279,12 +279,12 @@ module Registry {
             tab.remove(sre.name);
         }
 
-        proc unregister_bitvector(bre: borrowed BitVectorRegEntry) throws {
+        proc unregister_bitvector(bre: shared BitVectorRegEntry) throws {
             registered_entries.remove(bre.array);
             tab.remove(bre.name);
         }
 
-        proc lookup(name: string): borrowed AbstractRegEntry throws {
+        proc lookup(name: string): shared AbstractRegEntry throws {
             checkTable(name, "lookup");
             return tab[name];
```

Comment:
To resolve the CI build issues, I would replace the `return tab[name]` with `table.table[slot].val`. This touches some internals that aren't really user-facing, but, in my opinion, is the most simple workaround here.

Reply:
For context, we have to do this because there was a bug in Chapel 1.30 with the Map module (that I introduced, it was all my fault 😅)